test/integration/test_create.py

```diff
@@ -50,3 +52,81 @@ def test_create_streams_output_with_invalid_verbosity(cli, build_dir_and_ee_yml)
     result = cli(f"ansible-builder create -c {tmpdir} -f {eeyml} -v 6", allow_error=True)
     assert result.rc != 0
     assert 'invalid choice: 6 (choose from 0, 1, 2, 3)' in (result.stdout + result.stderr)
+
+
+def test_collection_verification_off(cli, build_dir_and_ee_yml):
+    """
+    Test that, by default, collection verification is off via the env var.
+    """
+    ee = [
+        'dependencies:',
+        '  galaxy: requirements.yml',
+    ]
+    req = [
+        'collections:',
+        '  - name: community.general',
+    ]
+    tmpdir, eeyml = build_dir_and_ee_yml("\n".join(ee))
+    reqyml = tmpdir / "requirements.yml"
+    reqyml.write_text("\n".join(req))
+    cli(f'ansible-builder create -c {tmpdir} -f {eeyml} --output-filename Containerfile')
+
+    containerfile = tmpdir / "Containerfile"
+    assert containerfile.exists()
+    assert "RUN ANSIBLE_GALAXY_DISABLE_GPG_VERIFY=1 ansible-galaxy" in containerfile.read_text()
+
+
+def test_collection_verification_on(cli, build_dir_and_ee_yml):
+    """
+    Test that collection verification is on when given a keyring.
+    """
+    ee = [
+        'dependencies:',
+        '  galaxy: requirements.yml',
+    ]
+    req = [
+        'collections:',
+        '  - name: community.general',
+    ]
+    tmpdir, eeyml = build_dir_and_ee_yml("\n".join(ee))
+    reqyml = tmpdir / "requirements.yml"
+    reqyml.write_text("\n".join(req))
+    keyring = tmpdir / "mykeyring.gpg"
+    keyring.touch()
+    cli(f'ansible-builder create -c {tmpdir} -f {eeyml} --output-filename Containerfile --galaxy-keyring {keyring}')
+
+    containerfile = tmpdir / "Containerfile"
+    assert containerfile.exists()
+    text = containerfile.read_text()
+
+    keyring_copy = tmpdir / constants.user_content_subfolder / constants.default_keyring_name
+    assert keyring_copy.exists()
+
+    assert "RUN ANSIBLE_GALAXY_DISABLE_GPG_VERIFY=1 ansible-galaxy" not in text
+    assert f"--galaxy-keyring {constants.default_keyring_name}" in text
+
+
+def test_galaxy_signing_extra_args(cli, build_dir_and_ee_yml):
+    """
+    Test that all extr asigning args for gpg are passed into the container file.
+    """
+    ee = [
+        'dependencies:',
+        '  galaxy: requirements.yml',
+    ]
+    req = [
+        'collections:',
+        '  - name: community.general',
+    ]
+    tmpdir, eeyml = build_dir_and_ee_yml("\n".join(ee))
+    reqyml = tmpdir / "requirements.yml"
+    reqyml.write_text("\n".join(req))
+    cli(f'ansible-builder create -c {tmpdir} -f {eeyml} --output-filename Containerfile '
+        '--galaxy-ignore-signature-status-code 500 --galaxy-required-valid-signature-count 3')
```

Comment:
Are these options valid without specifying a keyring? If not, we probably should have a check for that early on in builder that errors out quickly for the user.